raiden-ts/src/services/epics.ts

```diff
@@ -1193,7 +1197,7 @@ function parsePfsResponse(
   pfsResponse: Response,
   responseText: string,
   iou: Signed<IOU> | undefined,
-  pfsSafetyMargin: number,
+  { pfsSafetyMargin, pfsMaxPaths }: RaidenConfig,
```

Comment:
Why is the type picking not necessary here? I lack in understanding when to use it and when not. I don't see the difference to change two lines above (within this PR). Thanks for explaining it in advanced! :pray:

Reply:
It isn't needed because it's the `RaidenConfig` object.  I usually don't pick config values, since the config may be expected to be passed as a whole. I tend to pick only `deps` values, and only on local/non-exported utilities, so both be able to pass only the subset, and have an error if I try to use anything else. On configs and other destructured values, I don't think these constraints are needed, but of course it's up to the implementer.

Reply:
Ah okay so the Pick is actually preventing the caller to pass the whole object. The other one would simply disassamble it.

Reply:
It doesn't prevent, just doesn't require. If you don't declare with `Pick`, even if you destructure a single value, you are required to pass the whole object by interface. By `Pick`ing, you can pass the whole object, or you can simply do something like `{ log }` (if the interface only requires `log`).